AtalaPrismSDK/Pollux/Sources/PolluxImpl+CredentialRequest.swift

```diff
@@ -17,18 +17,23 @@ extension PolluxImpl {
             throw PolluxError.offerDoesntProvideEnoughInformation
         }
         
-        switch offerAttachment.mediaType {
+        switch offerAttachment.format {
         case "jwt", "", .none:
```

Comment:
just checking has this been tested? I thought the jwt cred-offer had a format of "prism/jwt"

Reply:
This was a nice catch. I actually tested and worked on this morning but it didn't went on the commit my mistake. Updated now its solved ;)